.github/workflows/main.yml

```diff
@@ -100,33 +109,35 @@ jobs:
         token: ${{ secrets.GITHUB_TOKEN }}
     - name: Zip Test Output (opengles)
       run: |
-        7z a -tzip test-output-linux-opengles.zip love2d-${{ github.sha }}/testing/output/
+        7z a -tzip test-output-linux-opengles-${{ runner.arch }}.zip love2d-${{ github.sha }}/testing/output/
     - name: Artifact Test Output (opengles)
       uses: actions/upload-artifact@v4
       with:
-        name: test-output-linux-opengles-${{ steps.report2.outputs.conclusion }}
-        path: test-output-linux-opengles.zip
+        name: test-output-linux-opengles-${{ runner.arch }}-${{ steps.report2.outputs.conclusion }}
+        path: test-output-linux-opengles-${{ runner.arch }}.zip
     # linux vulkan tests
     - name: Run Test Suite (vulkan)
+      if: runner.arch != 'ARM64'
```

Comment:
Can we make it test vulkan? Debug mode doesn't strictly need to be enabled, although it would be nice.

Reply:
No problem. Enabled

Reply:
The vulkan linux arm64 test is failing to start because debug mode is enabled there, which requires the vulkan SDK. So you can either disable the graphics debug environment variable on arm64 linux, or make it build the vulkan SDK.

Reply:
added for vulkan
`LOVE_GRAPHICS_DEBUG: ${{ runner.arch == 'ARM64' && '0' || '1' }}`